TESTS/mbed_hal_fpga_ci_test_shield/uart/main.cpp

```diff
@@ -338,7 +338,9 @@ Case cases[] = {
     Case("38400, 8N1, FC off", one_peripheral<UARTNoFCPort, DefaultFormFactor, fpga_uart_test_common_no_fc<38400, 8, ParityNone, 1, false> >),
     Case("115200, 8N1, FC off", one_peripheral<UARTNoFCPort, DefaultFormFactor, fpga_uart_test_common_no_fc<115200, 8, ParityNone, 1, false> >),
     // stop bits
+#if !defined(TARGET_NRF52840)
```

Comment:
i really don't like this per board exceptions. I'd rather have a macro defined in the target definition, something like UART_WHATEVER_NOT_SUPPORTED. It's way clearer and also gives you a clear answer to "why".

Reply:
Like I mentioned in the description I consider this as a temporary solution that should be replaced by `get_capabilities()` function when @fkjagodzinski is back. 
But using `UART_WHATEVER_NOT_SUPPORTED` should be easy to change. I will do it later today.

Reply:
Fixed